src/HasSlug.php

```diff
@@ -52,6 +52,17 @@ public function generateSlug()
         $this->addSlug();
     }
 
+    public function resolveRouteBinding($value)
+    {
+        if ($this->getSlugOptions()->routeBinding) {
+            return $this->where($this->getRouteKeyName(), $value)
+                        ->orWhere($this->getSlugOptions()->slugField, $value)
+                        ->first();
+        } else {
+            return parent::resolveRouteBinding($value);
```

Comment:
Could you move this out of the `if` construct, so we can use the `else`branch. Early returns ftw!

Reply:
Hey Freek, I was not exactly sure what you meant by

> so we can use the `else` branch

So in my most recent commit (https://github.com/spatie/laravel-sluggable/pull/115/commits/72b512ede88d4252fc9336207a6e8598a98f5cc9) I just removed the else (so if it was what you meant there yah go). Please let me know if this is not what you are referring to.